protocol/cors/accept-acah.feature

```diff
@@ -0,0 +1,16 @@
+# This only applies to pre-flight checks since that is where Access-Control-Allow-Headers is used
+Feature: Server should explicitly list Accept under Access-Control-Allow-Headers
```

Comment:
Well, this is only a SHOULD in the spec.

I think we should test the case where a clients sends an Accept header that is [longer than 128 characters](https://fetch.spec.whatwg.org/#ref-for-byte-sequence-length), which is easy if you basically list all RDF types with a quality value. Then `Accept` is no longer whitelisted, and MUST be explicitly allowed.

I think in that case, the browser would explicitly list `Accept` in `Access-Control-Request-Headers` for `OPTIONS`. But we need to check the `GET` case too.

Reply:
https://fetch.spec.whatwg.org/#cors-preflight-fetch mentions `Accept: */*` in step 2 (and I can see that being included in actual requests). Besides that, I agree with the understand of Accept in ACRH ACAH..